python/paddle/jit/dy2static/program_translator.py

```diff
@@ -1691,9 +1693,21 @@ def func(x):
 
 
 @switch_to_static_graph
-def _to_prim(blocks, blacklist=frozenset(), whitelist=frozenset()):
+def _to_prim(
+    blocks,
+    blacklist=frozenset(),
+    whitelist=frozenset(),
```

Comment:
Do we need it?

Reply:
To avoid circular dependency. And this will be deleted later.